File: ios/ObjectiveC/Beacon/3-Alert/Complete/beaconAlertComplete/AlertViewControllerAction.m
//
//  AlertViewControllerAction.m
//  beaconAlertComplete
//
//  Created by sarra srairi on 08/04/2016.
//  Copyright © 2016 sarra srairi. All rights reserved.
//

#import "AlertViewControllerAction.h"

@interface AlertViewControllerAction (){
 
}

@end

@implementation AlertViewControllerAction

@synthesize currentPlaceInAppAction;

- (void)viewDidLoad {
    [super viewDidLoad];
    self.txtdescription.text = [currentPlaceInAppAction getDescription];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: ios/SWIFT/Beacon/3-Alert/Starter/AlertStrategy/AlertViewControllerAction.swift
//
//  AlertViewControllerAction.swift
//  AlertStrategy
//
//  Created by Ludovic Vimont on 20/11/2017.
//  Copyright © 2017 R&D connecthings. All rights reserved.
//

import Foundation
import UIKit
import AdtagConnection
import AdtagLocationBeacon

public class AlertViewControllerAction: UIViewController {
    @IBOutlet weak var textDescription: UILabel!
    var currentPlaceInAppAction: AdtagPlaceInAppAction!
    
    public override func viewDidLoad() {
        super.viewDidLoad()
        textDescription.text = currentPlaceInAppAction.getDescription()
        AdtagBeaconManager.shared.redirect(placeInAppAction: currentPlaceInAppAction, from: "AlertViewControllerAction")
    }

    public override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}
